cloud/shared/validate_variable_definitions.py

```diff
@@ -0,0 +1,76 @@
+import os
+import glob
+import json
+
+class ValidateVariableDefinitions():
+    def __init__(self, variable_definitions = {}):
+        self.variable_definitions = variable_definitions
+
+    def load_repo_variable_definitions_files(self):
+        self.variable_definitions = {}
+        cwd = os.getcwd()
+
+        definition_file_paths = glob.glob(cwd + '/cloud/**/variable_definitions.json', recursive=True)
```

Comment:
Do you expect that there will be more than one `variable_definitions.json` file in for a single cloud provider, or will we define one for each cloud we're deploying to (plus a shared)? Staging and Prod environments should require the same variables, just with different values, right?

Reply:
My guess is we'll have one shared and then one for each provider. The glob here might be a little silly and we should just load them each explicitly as we add them.

> Staging and Prod environments should require the same variables, just with different values, right?

yep!